### PR TITLE
Add Dockerfile for rangda

### PR DESCRIPTION
**Pull request by alinetskyi**

### CODE REVIEW

File: cmd/rangda/db_controller.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/go-yaml/yaml"
+	"github.com/jmoiron/sqlx"
+	"github.com/kovetskiy/ko"
+)
+
+type db_credentials struct {
+	Host    string `env:"DATABASE_HOST" default:"localhost"`
+	User    string `env:"DATABASE_USER" default:"root"`
+	Pwd     string `env:"DATABASE_PASSWORD" default:""`
```
Comment (kovetskiy) [inline cmd/rangda/db_controller.go]: Pwd -> Password

File: cmd/rangda/db_controller.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	DB_name string `env:"DATABASE_NAME" default:"barong_development"`
```
Comment (kovetskiy) [inline cmd/rangda/db_controller.go]: Name

File: cmd/rangda/db_controller.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+func make_db_address(conf db_credentials) string {
+	conn_addr := conf.User + ":" + conf.Pwd + "@tcp(" + conf.Host + ":3306)" + conf.DB_name
+	return conn_addr
+}
+
+func connect_to_db() bool {
```
Comment (kovetskiy) [inline cmd/rangda/db_controller.go]: such function should return pointer on *sql.DB and error, not bool

File: cmd/rangda/db_controller.go [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	var config db_credentials
+	var db *sqlx.DB
+	var err = ko.Load("database.yaml", &config, yaml.Unmarshal)
+
+	if err != nil {
+		panic(err)
```
Comment (kovetskiy) [inline cmd/rangda/db_controller.go]: panic should be used only in emergency cases, missing config file or broken db connection is not emergency case. 

Emergency case it's when the program can't run anymore, for example, nil dereference is real case when you should panic.

**Review by kovetskiy — CHANGES_REQUESTED** [review]

**Comment by seletskiy** [comment]
@alinetskyi: hey, make sure to read [common changes requested in Go code during PRs][1].

Relevant links to this PR are: [don't panic][2] and [use mixed caps][3].

[1]: https://github.com/golang/go/wiki/CodeReviewComments
[2]: https://github.com/golang/go/wiki/CodeReviewComments#dont-panic
[3]: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

**Pull request closed without merging**